app/assets/javascripts/sitewide/tiny_mce.js

```diff
@@ -226,17 +287,18 @@ var TinyMCE = (function() {
             });
 
             editor.on('Dirty', function() {
-              var editorForm = $(editor.getContainer()).closest('form');
-              editorForm.find('.tinymce-status-badge').addClass('hidden');
-              $(editor.getContainer())
-                .find('.tinymce-save-button').removeClass('hidden');
+              refreshSaveButton(editor);
```

Comment:
I already made it here - https://github.com/biosistemika/scinote-web/pull/2082

Reply:
Well this is on `master` branch and yours is on `activestorage_migration`. It should be fixed when merging.

Reply:
Ok

Reply:
I renamed my function to `makeItDirty` to easen the merge process.